google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/models/Mutation.java

```diff
@@ -43,18 +44,42 @@
   static final int MAX_MUTATIONS = 100_000;
   @InternalApi("Visible for testing")
   static final int MAX_BYTE_SIZE = 200 * 1024 * 1024;
+  @InternalApi("Visible for testing")
+  static final long SERVER_SIDE_TIMESTAMP = -1;
+
+  private final boolean allowServersideTimestamp;
 
   private transient ImmutableList.Builder<com.google.bigtable.v2.Mutation> mutations =
       ImmutableList.builder();
 
   private int numMutations;
   private long byteSize;
 
+  /**
+   * Creates new instance of Mutation object.
+   *
+   * @return Mutation object.
```

Comment:
I've been avoiding adding `@return` & `@param` annotations unless they provide any addition info. Please remove it here and below

Reply:
done. Thanks.